index.js

```diff
@@ -211,23 +213,37 @@ module.exports = () => {
 		};
 	};
 
+	const drainOpenSubscriptions = async () => {
+		let i = openSubscriptions.length;
+		while (i--) {
+			debug('Closing active subscription');
+			await openSubscriptions[i].close();
+			openSubscriptions.splice(i, 1);
+		}
```

Comment:
why not just a promise.all and delete all of the at the same time? Something like:
`await Promise.all(openSubscriptions.map(async sub => await sub.close()))`

Reply:
Could do. Went this route to enable the debug statement as per the rest of the module design.